app/admin/dashboard.rb

```diff
@@ -0,0 +1,33 @@
+ActiveAdmin.register_page "Dashboard" do
+
+  menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    div class: "blank_slate_container", id: "dashboard_default_message" do
+      span class: "blank_slate" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.